addons/website/static/src/js/menu/edit.js

```diff
@@ -132,11 +132,10 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
      * @private
      */
     _addEditorMessages: function () {
-        var $target = this._targetForEdition();
-        this.$editorMessageElements = $target
-            .find('.oe_structure.oe_empty, [data-oe-type="html"]')
-            .not('[data-editor-message]')
-            .attr('data-editor-message', _t('DRAG BUILDING BLOCKS HERE'));
+        const $target = this._targetForEdition();
+        this.$editorMessageElements = $target.find('.oe_structure.oe_empty, [data-oe-type="html"]');
+        this.$editorMessageElements.not('[data-editor-message]').attr('data-editor-message', _t('DRAG BUILDING BLOCKS HERE'));
```

Comment:
`this.$editorMessageElements` did not contain the elements which already have data-editor-message attribute, after your commit it does. Is it not a problem?

Reply:
I have done some testing and the affected code has no regression.
`removeAttr` does not throw an exception if the attribute is not found.

Reply:
Who talked about `removeAttr`? The point is that you are changing the content of the `this.$editorMessageElements` variable in a stable version, so any custo relying on it will potentially be broken. There is no reason to risk that that I could see here.